src/index.js

```diff
@@ -92,6 +105,59 @@ function determineTrip(event) {
     return +event.target.id.split('-')[1] === trip.id
   })
   if (trip) {
-    domUpdates.displayTripDetails(trip)
+    domUpdates.displayTripDetails(trip, false)
   }
 }
+function getDestinationsList() {
+  let destinationList = destinationsData.reduce((list, destination) => {
+    if (!list.includes(destination.destination)) {
+      list.push(destination.destination)
```

Comment:
Consider renaming property `destination` on object `destination`. Is it a name<string>? Perhaps `destinationName` if so...

Reply:
easy fix would be to rename the parameter in the reduce function - `destinationDatum` or similar to avoid `destination.destination`